src/App.jsx

```diff
@@ -1,35 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from 'react';
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  {
+    id: 1,
+    title: "Commit project and create working branch"
+  },
+  {
+    id: 2,
+    title: "Create the beginnings of the todo list"
+  },
+  {
+    id: 3,
+    title: "Complete assignment"
+  }
+];
+
+function listItem (item) {
```

Comment:
I like the instinct to pull this part out. I would tweak the name here to be capitalized, so `ListItem`. We'll see this in upcoming lessons, but this is a React component, which follows a capital naming convention.